
#include <cerrno>

#include <exercises/errors-handling/Exceptions.hpp>

Exceptions::Exceptions()
{
}

Exceptions::Exceptions(const Exceptions& orig)
{
}

Exceptions::~Exceptions()
{
}

